src/coreclr/System.Private.CoreLib/src/Internal/Runtime/InteropServices/ComActivator.cs

```diff
@@ -627,6 +627,9 @@ internal sealed class LicenseInteropProxy
         private object? _licContext;
         private Type? _targetRcwType;
 
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2111",
```

Comment:
Is there a short name we can give to `IL2111`? That way people don't have to go look up what `IL2111` is? Like what we do with `IL2026` - `IL2026:RequiresUnreferencedCode`

Reply:
I can't come up with a great short name - my best idea at the moment is "ReflectionAccessToMemberWithDynamicallyAccessedMembersRequirements". 😦 @vitek-karas do you have any ideas?

Maybe we can just call it "RequiresDynamicallyAccessedMembers" which will be ambiguous with other warning codes (those about direct calls to methods with DAM requirements). We already have a similar ambiguity with RUC since IL2026 is used both for direct calls to RUC methods, and for reflection access to RUC methods.

Reply:
How about `IL2111:ReflectionToDynamicallyAccessedMembers`?